apps/package.json

```diff
@@ -88,6 +89,7 @@
     "node-sass": "1.2.2",
     "npm-which": "2.0.0",
     "phantomjs-prebuilt": "^2.1.3",
+    "radium": "^0.17.1",
```

Comment:
Exactly. It would probably be nice if we could have test only modules live in tests/package.json, but I don't think npm will go recurse into subdirectories to do installs.